[2024-05-02T17:11:39.450+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: diabetes_prediction.fetch_dataset manual__2024-05-02T17:11:33.568006+00:00 [queued]>
[2024-05-02T17:11:39.475+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: diabetes_prediction.fetch_dataset manual__2024-05-02T17:11:33.568006+00:00 [queued]>
[2024-05-02T17:11:39.476+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2024-05-02T17:11:39.525+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): fetch_dataset> on 2024-05-02 17:11:33.568006+00:00
[2024-05-02T17:11:39.538+0000] {standard_task_runner.py:57} INFO - Started process 93 to run task
[2024-05-02T17:11:39.546+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'diabetes_prediction', 'fetch_dataset', 'manual__2024-05-02T17:11:33.568006+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpkxm_e8_e']
[2024-05-02T17:11:39.551+0000] {standard_task_runner.py:85} INFO - Job 121: Subtask fetch_dataset
[2024-05-02T17:11:39.728+0000] {task_command.py:410} INFO - Running <TaskInstance: diabetes_prediction.fetch_dataset manual__2024-05-02T17:11:33.568006+00:00 [running]> on host 8baaf4bc95aa
[2024-05-02T17:11:40.026+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='diabetes_prediction' AIRFLOW_CTX_TASK_ID='fetch_dataset' AIRFLOW_CTX_EXECUTION_DATE='2024-05-02T17:11:33.568006+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-02T17:11:33.568006+00:00'
[2024-05-02T17:11:41.914+0000] {warnings.py:110} WARNING - /opt/***/dags/dag1.py:14: DtypeWarning: Columns (10) have mixed types.Specify dtype option on import or set low_memory=False.
  data = fetch_ucirepo(id=296)

[2024-05-02T17:11:42.056+0000] {warnings.py:110} WARNING - /opt/***/dags/dag1.py:18: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df['readmitted'] = y.readmitted

[2024-05-02T17:11:43.229+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-05-02T17:11:43.264+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=diabetes_prediction, task_id=fetch_dataset, execution_date=20240502T171133, start_date=20240502T171139, end_date=20240502T171143
[2024-05-02T17:11:43.328+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2024-05-02T17:11:43.415+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
